internal/callbacker/callbacker.go

```diff
@@ -4,7 +4,7 @@ import (
 	"time"
 )
 
-type CallbackerI interface {
+type SendCallbackInterface interface {
```

Comment:
I see nothing wrong with `CallbackerI`. We could decide on one standard for naming interfaces, but having to call them differently than the package name is too much of an restriction IMO.

Reply:
Changed to SenderI